src/cljs/athens/parse_renderer.cljs

```diff
@@ -58,18 +57,24 @@
                                          :cursor "alias"}]]})
 
 ;;; Helper functions for recursive link rendering
+(defn get-block-node-from-string
+  "Gets a block's node from the display string name (or partially parsed string tree)"
```

Comment:
I don't think a "block's node" is the right terminology. A node is a block. I'd call this `get-node-from-string` or `pull-node`.